src/commands/openYAMLConfigFile.ts

```diff
@@ -47,7 +47,8 @@ export async function openYAMLConfigFile(context: IActionContext, node?: StaticW
         }
     }
 
-    const configDocument: TextDocument = await workspace.openTextDocument(yamlFileUri);
+    // temporariy workaround for https://github.com/microsoft/vscode/issues/131925
+    const configDocument: TextDocument = process.platform === 'win32' ? await workspace.openTextDocument(yamlFileUri.fsPath) : await workspace.openTextDocument(yamlFileUri);
```

Comment:
Can you try using `Uri.file` above instead of `Uri.parse`? I think that's the root cause of the problem.